lib/providers/google.js

```diff
@@ -10,13 +10,16 @@ exports = module.exports = function (options) {
         scope: ['profile', 'email'],
         profile: function (credentials, params, get, callback) {
 
-            get('https://www.googleapis.com/plus/v1/people/me', null, (profile) => {
+            get('https://www.googleapis.com/oauth2/v3/userinfo', null, (profile) => {
 
                 credentials.profile = {
                     id: profile.id,
-                    displayName: profile.displayName,
-                    name: profile.name,
-                    emails: profile.emails,
+                    displayName: profile.name,
+                    name: {
+                        given_name: profile.given_name,
+                        family_name: profile.family_name
```

Comment:
We unfortunately don't have proper conventions enforced through all the providers so I don't mind having it yet again different if it means a bit less breakage.